### PR TITLE
Update to endpoint to delete favorites(products) of a user with error handlers/responses

### LINKED ISSUE #194: Update the Endpoint to delete favorites(products) of a user with error handlers/responses.
## User story
**Who:** **As a** developer
**What:** **I want to** to  be able to get the response status codes correctly
**Why:** **so that I can** see the difference between the correct and wrong entries of the query

## Acceptance criteria
If the ((user_id && product_id)===number)=> it should show the result and status code:200,
If the ((user_id || product_id) || (user_id && product_id) ===NaN || null)=> it should show an error message with status code other than 200.

## Implementation Details
Update the Delete endpoint for `/api/favorites/delete` with the required responses.

Read about Swagger here https://github.com/HackYourFuture-CPH/fp-class18/#-working-with-api-documentation---swagger and this article should also help you with that https://blog.logrocket.com/documenting-your-express-api-with-swagger/

### PR DESCRIPTION
**Pull request by nataliasowa**
# Description

Setting the response status codes correctly in the controller.

Fixes #194 

# How to test?
npm run db:setup
npm run dev
http://localhost:3000/api/documentation/#/  --> SWAGER

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by saloumeh-67 — APPROVED** [review]
i opened it in swagger and check it in DB it all works :).

File: src/server/api/controllers/favorites.controller.js [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
 
-const addToFavorites = async (body) => {
-  console.log('in add');
-  await knex('favorites').insert({
-    user_id: body.user_id,
-    product_id: body.product_id,
-    created_at: moment(Date.now()).format(),
-  });
+const addToFavorites = async(body) => {
+    console.log('in add');
```
Comment (dpfernandes) [inline src/server/api/controllers/favorites.controller.js]: Please remove console log

**Review by dpfernandes — CHANGES_REQUESTED** [review]

**Review by Divyajg — COMMENTED** [review]
Code seems as it would work, I cannot test it in my machine as I have slightly changes DB.

**Review by m-elshawa — APPROVED** [review]
this is what shows up in case of succes. in case id is not a number it doesn't allow it, and in case of too big numbers it give the same message. Thus I'm not sure if this is how it should work. (error message don't come up)
image

**Review by miltonhultgren — APPROVED** [review]
👍🏼

**Pull request merged**